x/evm/keeper/state_transition.go

```diff
@@ -423,7 +423,8 @@ func (k *Keeper) ApplyMessageWithConfig(ctx sdk.Context, msg core.Message, trace
 	// for more info https://github.com/tharsis/ethermint/issues/1085
 	// NOTE: MinGasDenominator can not be negative as it is validated on ValidateParams
 	gasLimit := sdk.NewDec(int64(msg.Gas()))
-	minimumGasUsed := gasLimit.Mul(cfg.Params.MinGasMultiplier)
+	minGasMultiplier := k.GetMinGasMultiplier(ctx)
```

Comment:
just as a note, we should doublecheck that this multiplier cannot be `0` as it could make the transaction error (by transferring 0 coins using the bank module)